pkg/controller/signup.go

```diff
@@ -46,6 +47,51 @@ func (s *Signup) PostHandler(ctx *gin.Context) {
 	ctx.Writer.WriteHeaderNow()
 }
 
+// UpdateVerificationHandler stars the verification process and updates a usersignup resource
+func (s *Signup) UpdateVerificationHandler(ctx *gin.Context) {
+	userID := ctx.GetString(context.SubKey)
+	signup, err := s.signupService.GetUserSignup(userID)
+	if err != nil {
+		if errors2.IsNotFound(err) {
+			log.Error(ctx, err, "usersignup not found")
+			errors.AbortWithError(ctx, http.StatusNotFound, err, "usersignup not found")
+		}
+		log.Error(ctx, err, "error retreiving usersignup")
```

Comment:
```suggestion
		log.Error(ctx, err, "error retrieving usersignup")
```

Reply:
fixed as part of: 60ca7377aabe427ed8f8e3d642380b1b8396454e